frontend/src/bundles/common/enums/app-route.enum.ts

```diff
@@ -9,6 +9,7 @@ const AppRoute = {
     EMPLOYER_ONBOARDING: '/employer/onboarding',
     TALENT_STEP: '/talent/onboarding/step/:step',
     NOT_FOUND: 'not-found',
+    MY_PROFILE: '/my/:step',
```

Comment:
Maybe it's better to write 'my-cabinet' or 'my-card'

Reply:
@nika-sirak maybe
I've chosen my/.. as in djinni

Reply:
`my-profile` will be good